mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -205,6 +205,8 @@
     <string name="browser_menu_manage_extensions">Manage extensions</string>
     <!-- Browser menu button that opens AMO in a tab -->
     <string name="browser_menu_discover_more_extensions">Discover more extensions</string>
+    <!-- Browser menu description that is shown when extensions are disabled due to extension errors -->
+    <string name="browser_menu_extensions_disabled_description">Temporarily disabled</string>
```

Comment:
@gabrielluong Please update comment to something like "Browser menu description that is shown when **one or more** extensions are disabled due to extension errors" ([previous comment](https://github.com/mozilla-l10n/android-l10n/pull/791#discussion_r1708414254)) 
This will help me finalize expsoing the remaining strings in this PR.
Thanks!

Reply:
@gabrielluong Ping in case this fell through the cracks